tests/test_utils.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import tempfile
+import unittest
+
+import pandas as pd
+
+from lsst.ap.association.utils import readSchemaFromApdb, make_empty_catalog, convertTableToSdmSchema
+import lsst.dax.apdb as daxApdb
+
+
+class TestUtils(unittest.TestCase):
+
+    def test_make_empty_catalog(self):
+        """Check that an empty catalog has the correct format.
+        """
+        apdb_config = daxApdb.ApdbSql.init_database(db_url="sqlite://")
+        config_file = tempfile.NamedTemporaryFile()
+        self.addCleanup(config_file.close)
+        apdb_config.save(config_file.name)
+        apdb = daxApdb.Apdb.from_uri(config_file.name)
```

Comment:
When writing this kind of code, I recommend reading [the API docs](https://pipelines.lsst.io/v/daily/py-api/lsst.dax.apdb.Apdb.html) instead of cargo-culting:
```suggestion
        apdb = daxApdb.Apdb.from_config(apdb_config)
```